tests/Sdk/Unit/Core/Application/Service/ManifestNormalizer/ManifestNormalizerRegistryTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Unit\Core\Application\Service\ManifestValidation;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Sdk\Core\Application\Dependency\ManifestConfigurationInterface;
+use SprykerSdk\Sdk\Core\Application\Exception\ManifestValidatorMissingException;
+use SprykerSdk\Sdk\Core\Application\Service\ManifestValidator\ManifestValidatorRegistry;
+
+/**
+ * @group Sdk
+ * @group Core
+ * @group Application
+ * @group Service
+ * @group ManifestValidation
+ * @group ManifestValidatorFactoryTest
+ */
+class ManifestNormalizerRegistryTest extends Unit
+{
+    /**
+     * @return void
+     */
+    public function testResolve(): void
+    {
+        // Arrange
+        $manifestValidator = $this->createMock(ManifestConfigurationInterface::class);
+        $manifestValidator->expects($this->once())
+            ->method('getName')
+            ->willReturn('task');
+
+        $manifestValidatorFactory = new ManifestValidatorRegistry([$manifestValidator]);
+
+        // Act
+        $result = $manifestValidatorFactory->resolve('task');
+
+        // Assert
+        $this->assertSame($manifestValidator, $result);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCanNotResolve(): void
```

Comment:
please the following pattern for the test methods - `test<methodName><someConditions>`